apps/airthings/airthings.star

```diff
@@ -235,19 +242,19 @@ def get_schema():
             schema.Text(
                 id = "clientSecret",
                 name = "AirThings API Client Secret",
-                desc = "API secret from https://dashboard.airthings.com/integrations/api-integration",
+                desc = "REQUIRED: API secret from https://dashboard.airthings.com/integrations/api-integration",
```

Comment:
Note that these field descriptions strings are not visible to the end user in the iOS/Android app.